### DISCUSSION TITLE
Where CMAKE_BUILD_TYPE is set?

### DISCUSSION THREAD

**Discussion by shiltian** [microsoft/vscode-cmake-tools#2997] [Q&A] [answered]
👋 Hi there,
I'm wondering where `CMAKE_BUILD_TYPE` is set? Currently I can see it automatically adds `-DCMAKE_BUILD_TYPE:STRING=Debug` in the command, but I don't see where I can configure it. I tried to add the following setting in my workspace `settings.json`:
```
  "cmake.configureSettings": {
    "CMAKE_BUILD_TYPE": "RelWithDebInfo",
  },
```
But I still see `-DCMAKE_BUILD_TYPE:STRING=Debug` in the command. Is there anyway I can change it?

**Comment by bobbrow** [comment, marked as answer]
It is set either with the "CMake: Select Variant" command, or clicking on the item in the bottom left of the status bar:  ![image](https://user-images.githubusercontent.com/12818240/216649030-471c4800-c0ed-40cf-bf46-b686d84ccbf1.png)

**Reply by shiltian** [reply]
I see. Thanks for the info!

**Reply by skyleaworlder** [reply]
Thank you! You are my hero.
